frontend/src/modules/root/components/Navbar/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { CloseOutlined, MenuOutlined } from '@ant-design/icons';
+import { Dropdown } from 'antd';
+import Container from 'common/components/Container';
+import React, { useState } from 'react';
+import NavbarMenu from './components/NavbarMenu';
+import {
+  Department,
+  Faculty,
+  LogoCPE,
+  LogoKMUTT,
+  MenuButton,
+  Nav,
+  NavbarBrand,
+  StyledDivider,
+  UL,
+  Wrapper,
+} from './styled';
+
+export interface NavBarProps {
+  visible?: boolean;
+}
```

Comment:
mvoe this line to another file `interface.ts` in the same `/components/Navbar/interface.ts`